centos-ci/README.md

```diff
@@ -0,0 +1,37 @@
+# LeApp CI
+
+LeApp CI is hosted on the [CentOS CI infrastructure](http://ci.centos.org/) using Jenkins and since
+our tests are virtualization heavy we have a dedicated slave node (slave03) for our tests.
+
+## Architecture
+
+When Jenkins is triggered via a webhook it uses a Duffy key located in `~/duffy.key` to get a node from the CICO
+cluster on which the test will run. Duffy is CentOS CI middleware for resource management inside the cluster.
+The lifetime (read, debuggability) of a failed test run is 12 hours.
```

Comment:
Any chance we could get that extended to 18 or 24 hours? 12 hours is cutting it fine for the "start a CI run at the end of the day, check it in the morning".

OTOH, I guess it encourages good test logging for post-mortem debugging, rather than expecting to be able to connect to the failed instance.